data/mm/world/overworld.yml

```diff
@@ -266,20 +268,58 @@
   exits:
     "Deku Palace": "true"
   locations:
-    "Deku Shrine Mask of Scents": "has(MASK_DEKU) && has(BOW)"
+    "Deku Shrine Mask of Scents": "true"
```

Comment:
You need to be able to hit the crystal at some point IIRC. Could account for hookshot/zora/deku bubble/etc tho.

Reply:
If I remember correctly, you can always jump as Link to the switch and hit it as Deku (that you're logically required to have to enter Deku Shrine anyway)

Reply:
Allright then

Reply:
Confirmed by MMR logic:
![image](https://user-images.githubusercontent.com/30158363/205110516-03190d4e-de12-4df8-a27d-f1ea6d5d1f9d.png)
![image](https://user-images.githubusercontent.com/30158363/205110565-7bd79dc0-4112-4558-a8eb-cc596a5d26f9.png)